showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITCommonServiceMixins.java

```diff
@@ -63,6 +65,16 @@ public void createClients() throws Exception {
     httpjsonClient = TestClientInitializer.createHttpJsonEchoClient();
   }
 
+  @After
+  public void destroyClients() throws InterruptedException {
+    grpcClient.close();
+    grpcClient.awaitTermination(TestClientInitializer.AWAIT_TERMINATION_SECONDS, TimeUnit.SECONDS);
+
+    httpjsonClient.close();
+    httpjsonClient.awaitTermination(
```

Comment:
This method is not implemented yet https://github.com/googleapis/gapic-generator-java/issues/1663